src/core/lib/security/security_connector/tls/tls_security_connector.cc

```diff
@@ -104,14 +105,14 @@ grpc_status_code TlsFetchKeyMaterials(
         gpr_log(GPR_DEBUG, "Credential does not change after reload.");
       } else if (arg->status == GRPC_SSL_CERTIFICATE_CONFIG_RELOAD_FAIL) {
         gpr_log(GPR_ERROR, "Credential reload failed with an error:");
-        if (arg->error_details != nullptr) {
-          gpr_log(GPR_ERROR, "%s", arg->error_details);
+        if (arg->error_details->has_error()) {
```

Comment:
`has_error` above seems to have a bug (true when error message is empty) which means this code was never exercised in a test?

Reply:
Yeah, Thanks for pointing this out! Currently we don't have the test for this. I will add a unit test for the class.

Reply:
fixed in the new PR.